autogpt_platform/backend/backend/blocks/rmfg/_http.py

```
@@ -0,0 +1,95 @@
+"""Transport helpers for the RMFG client: error envelopes and polling."""
+
+import asyncio
+import time
+from typing import Any, Awaitable, Callable, Optional, TypeVar
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Awaitable`, `Callable`

Import from `collections.abc`

(UP035)